Subject: [PATCH] feat: Update bindings to 26.1
---
Index: moe-core/moe.apple/moe.platform.ios/src/main/java/apple/gameplaykit/GKCompositeBehavior.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/gameplaykit/GKCompositeBehavior.java b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/gameplaykit/GKCompositeBehavior.java
--- a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/gameplaykit/GKCompositeBehavior.java	(date 1763056988743)
+++ b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/gameplaykit/GKCompositeBehavior.java	(revision 31dc980ebebd479461dac71d0f9c046638163878)
@@ -211,7 +211,7 @@
     @NotNull
     @Generated
     @Selector("objectAtIndexedSubscript:")
-    public native GKBehavior objectAtIndexedSubscript(@NUInt long idx);
+    public native GKGoal objectAtIndexedSubscript(@NUInt long idx);

     /**
      * Supports getting a weight via a [behavior] subscript.
Index: moe-core/moe.apple/moe.platform.ios/src/main/java/apple/uikit/UIContentUnavailableView.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/uikit/UIContentUnavailableView.java b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/uikit/UIContentUnavailableView.java
--- a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/uikit/UIContentUnavailableView.java	(date 1763056998004)
+++ b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/uikit/UIContentUnavailableView.java	(revision 31dc980ebebd479461dac71d0f9c046638163878)
@@ -417,7 +417,7 @@

     @Generated
     @Selector("setConfiguration:")
-    public native void setConfiguration(@NotNull UIContentUnavailableConfiguration value);
+    public native void setConfiguration(@NotNull UIContentConfiguration value);

     /**
      * Whether the content can scroll. Default is NO.
Index: moe-core/moe.apple/moe.platform.ios/src/main/java/apple/gamekit/protocol/GKLocalPlayerListener.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/gamekit/protocol/GKLocalPlayerListener.java b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/gamekit/protocol/GKLocalPlayerListener.java
--- a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/gamekit/protocol/GKLocalPlayerListener.java	(date 1763059388531)
+++ b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/gamekit/protocol/GKLocalPlayerListener.java	(revision 31dc980ebebd479461dac71d0f9c046638163878)
@@ -16,11 +16,15 @@

 package apple.gamekit.protocol;

+import apple.foundation.NSArray;
+import apple.gamekit.GKPlayer;
 import org.moe.natj.general.ann.Generated;
 import org.moe.natj.general.ann.Library;
 import org.moe.natj.general.ann.Runtime;
 import org.moe.natj.objc.ObjCRuntime;
+import org.moe.natj.objc.ann.IsOptional;
 import org.moe.natj.objc.ann.ObjCProtocolName;
+import org.moe.natj.objc.ann.Selector;

 /**
  * API-Since: 7.0
@@ -31,4 +35,12 @@
 @ObjCProtocolName("GKLocalPlayerListener")
 public interface GKLocalPlayerListener extends GKChallengeListener, GKGameActivityListener, GKInviteEventListener,
         GKTurnBasedEventListener, GKSavedGameListener {
+
+    @Generated
+    @IsOptional
+    @Deprecated
+    @Selector("player:didRequestMatchWithPlayers:")
+    default void playerDidRequestMatchWithPlayers(GKPlayer player, NSArray<String> playerIDsToInvite) {
+        throw new java.lang.UnsupportedOperationException();
+    }
 }
Index: moe-core/moe.apple/moe.platform.ios/src/main/java/apple/foundation/NSMutableDictionary.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/foundation/NSMutableDictionary.java b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/foundation/NSMutableDictionary.java
--- a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/foundation/NSMutableDictionary.java	(date 1763059342243)
+++ b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/foundation/NSMutableDictionary.java	(revision 31dc980ebebd479461dac71d0f9c046638163878)
@@ -1057,11 +1057,4 @@
     @Selector("useStoredAccessor")
     public static native boolean useStoredAccessor();

-    /**
-     * Send -setObject:forKey: to the receiver, unless the value is nil, in which case send -removeObjectForKey:.
-     */
-    @Generated
-    @Selector("setValue:forKey:")
-    public native void setValueForKey(@Mapped(ObjCObjectMapper.class) @Nullable _ObjectType value, @NotNull String key);
-
 }
Index: moe-core/moe.apple/moe.platform.ios/src/main/java/apple/authenticationservices/ASAuthorizationSecurityKeyPublicKeyCredentialAssertionRequest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/authenticationservices/ASAuthorizationSecurityKeyPublicKeyCredentialAssertionRequest.java b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/authenticationservices/ASAuthorizationSecurityKeyPublicKeyCredentialAssertionRequest.java
--- a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/authenticationservices/ASAuthorizationSecurityKeyPublicKeyCredentialAssertionRequest.java	(date 1763059388528)
+++ b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/authenticationservices/ASAuthorizationSecurityKeyPublicKeyCredentialAssertionRequest.java	(revision 31dc980ebebd479461dac71d0f9c046638163878)
@@ -172,8 +172,7 @@

     @Generated
     @Selector("setAllowedCredentials:")
-    public native void setAllowedCredentials(
-            @NotNull NSArray<? extends ASAuthorizationSecurityKeyPublicKeyCredentialDescriptor> value);
+    public native void setAllowedCredentials(@NotNull NSArray<?> value);

     @Generated
     @Selector("setChallenge:")
Index: moe-core/moe.apple/moe.platform.ios/src/main/java/apple/callkit/CXCallDirectoryProvider.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/callkit/CXCallDirectoryProvider.java b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/callkit/CXCallDirectoryProvider.java
--- a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/callkit/CXCallDirectoryProvider.java	(date 1763059388530)
+++ b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/callkit/CXCallDirectoryProvider.java	(revision 31dc980ebebd479461dac71d0f9c046638163878)
@@ -18,6 +18,7 @@

 import apple.NSObject;
 import apple.foundation.NSArray;
+import apple.foundation.NSExtensionContext;
 import apple.foundation.NSMethodSignature;
 import apple.foundation.NSSet;
 import apple.foundation.protocol.NSExtensionRequestHandling;
@@ -160,7 +161,7 @@

     @Generated
     @Selector("beginRequestWithExtensionContext:")
-    public native void beginRequestWithExtensionContext(@NotNull CXCallDirectoryExtensionContext context);
+    public native void beginRequestWithExtensionContext(@NotNull NSExtensionContext context);

     @Generated
     @Selector("init")
Index: moe-core/moe.apple/moe.platform.ios/src/main/java/apple/foundation/NSDictionary.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/foundation/NSDictionary.java b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/foundation/NSDictionary.java
--- a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/foundation/NSDictionary.java	(date 1763056988035)
+++ b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/foundation/NSDictionary.java	(revision 31dc980ebebd479461dac71d0f9c046638163878)
@@ -280,7 +280,7 @@
     @Selector("countByEnumeratingWithState:objects:count:")
     @NUInt
     public native long countByEnumeratingWithStateObjectsCount(@NotNull VoidPtr state,
-            @NotNull @ReferenceInfo(type = ObjCObject.class) Ptr<_KeyType> buffer, @NUInt long len);
+            @NotNull @ReferenceInfo(type = ObjCObject.class) Ptr<ObjCObject> buffer, @NUInt long len);

     @NotNull
     @Generated
Index: moe-core/moe.apple/moe.platform.ios/src/main/java/apple/authenticationservices/ASAuthorizationPlatformPublicKeyCredentialAssertionRequest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/authenticationservices/ASAuthorizationPlatformPublicKeyCredentialAssertionRequest.java b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/authenticationservices/ASAuthorizationPlatformPublicKeyCredentialAssertionRequest.java
--- a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/authenticationservices/ASAuthorizationPlatformPublicKeyCredentialAssertionRequest.java	(date 1763059388526)
+++ b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/authenticationservices/ASAuthorizationPlatformPublicKeyCredentialAssertionRequest.java	(revision 31dc980ebebd479461dac71d0f9c046638163878)
@@ -172,8 +172,7 @@

     @Generated
     @Selector("setAllowedCredentials:")
-    public native void setAllowedCredentials(
-            @NotNull NSArray<? extends ASAuthorizationPlatformPublicKeyCredentialDescriptor> value);
+    public native void setAllowedCredentials(@NotNull NSArray<?> value);

     @Generated
     @Selector("setChallenge:")
Index: moe-core/moe.apple/moe.platform.ios/src/main/java/apple/metalperformanceshaders/MPSCNNConvolutionTranspose.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/metalperformanceshaders/MPSCNNConvolutionTranspose.java b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/metalperformanceshaders/MPSCNNConvolutionTranspose.java
--- a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/metalperformanceshaders/MPSCNNConvolutionTranspose.java	(date 1763059388536)
+++ b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/metalperformanceshaders/MPSCNNConvolutionTranspose.java	(revision 31dc980ebebd479461dac71d0f9c046638163878)
@@ -566,7 +566,7 @@
     @Generated
     @Selector("resultStateForSourceImage:sourceStates:destinationImage:")
     public native MPSCNNConvolutionTransposeGradientState resultStateForSourceImageSourceStatesDestinationImage(
-            @NotNull MPSImage sourceImage, @Nullable NSArray<? extends MPSCNNConvolutionGradientState> sourceStates,
+            @NotNull MPSImage sourceImage, @Nullable NSArray<? extends MPSState> sourceStates,
             @NotNull MPSImage destinationImage);

     /**
@@ -588,8 +588,7 @@
     @Selector("temporaryResultStateForCommandBuffer:sourceImage:sourceStates:destinationImage:")
     public native MPSCNNConvolutionTransposeGradientState temporaryResultStateForCommandBufferSourceImageSourceStatesDestinationImage(
             @NotNull @Mapped(ObjCObjectMapper.class) MTLCommandBuffer commandBuffer, @NotNull MPSImage sourceImage,
-            @Nullable NSArray<? extends MPSCNNConvolutionGradientState> sourceStates,
-            @NotNull MPSImage destinationImage);
+            @Nullable NSArray<? extends MPSState> sourceStates, @NotNull MPSImage destinationImage);

     @Generated
     @Deprecated
Index: moe-core/moe.apple/moe.platform.ios/src/main/java/apple/uikit/UIListContentView.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/uikit/UIListContentView.java b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/uikit/UIListContentView.java
--- a/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/uikit/UIListContentView.java	(date 1763056998230)
+++ b/moe-core/moe.apple/moe.platform.ios/src/main/java/apple/uikit/UIListContentView.java	(revision 31dc980ebebd479461dac71d0f9c046638163878)
@@ -238,7 +238,7 @@

     @Generated
     @Selector("setConfiguration:")
-    public native void setConfiguration(@NotNull UIListContentConfiguration value);
+    public native void setConfiguration(@NotNull UIContentConfiguration value);

     @Generated
     @Selector("debugDescription")
