diff --git a/include/plist/plist.h b/include/plist/plist.h
index c66545a..ce10411 100644
--- a/include/plist/plist.h
+++ b/include/plist/plist.h
@@ -85,12 +85,22 @@ extern "C"
     /**
      * The basic plist abstract data type.
      */
+#ifndef MOE_NATJGEN
     typedef void *plist_t;
+#else
+    typedef struct plist_private_t plist_private_t;
+    typedef plist_private_t *plist_t;
+#endif

     /**
      * The plist dictionary iterator.
      */
+#ifndef MOE_NATJGEN
     typedef void *plist_dict_iter;
+#else
+    typedef struct plist_dict_iter_private_t plist_dict_iter_private_t;
+    typedef plist_dict_iter_private_t *plist_dict_iter;
+#endif

     /**
      * The enumeration of plist node types.
